frontend/app/timed-admin/detail/controller.js

```diff
@@ -16,7 +16,13 @@ export default Controller.extend({
   loading: alias('fetchModels.isRunning'),
   project: alias('fetchModels.lastSuccessful.value.project'),
   orders: alias('fetchModels.lastSuccessful.value.orders'),
-  reports: alias('fetchModels.lastSuccessful.value.reports'),
+  reportsPage: 1,
+  reportsNext: false,
+
+  init() {
+    this._super(...arguments)
+    this.set('reports', [])
```

Comment:
Reports needs to be reset every time the route model changes. I think this way you have state which consists even if you navigate to another project.

Reply:
Same with `reportsPage` and `reportsNext`

Reply:
Moved to router controllerSetup hook